### PR TITLE
update Roslyn.Services.Editor.VisualBasic.UnitTests.dll unit tests to use localized resources

### PR DESCRIPTION
**Pull request by brettfo**
This also includes adding a resource for the string '&lt;result&gt;' that wasn't localized but should have been.

### CODE REVIEW

**Comment by Pilchie** [comment]
What string?

**Comment by Pilchie** [comment]
:shipit:

**Comment by brettfo** [comment]
Looks like I forgot to HTML-escape it :/
The missing string was &lt;result&gt;  See the last file in the changelist.

**Comment by brettfo** [comment]
@jasonmalinowski Since you initially triaged this bug to me, do you want to look at it?

**Comment by jasonmalinowski** [comment]
Nope. I only triaged it since @Pilchie was slacking at Codemash.

**Comment by brettfo** [comment]
Potential reviewers for second signoff: @rchande @dpoeschl @balajikris @basoundr

**Comment by rchande** [comment]
:+1:

File: src/EditorFeatures/VisualBasicTest/EndConstructGeneration/MethodBlockTests.vb [hunk 1]
```diff
@@ -56,10 +47,10 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.EndConstructGenera
                         "Class c1",
                         "    Sub New()",
                         "",
-                        "        ' This call is required by the designer.",
+                       $"        ' {ThisCallIsRequiredByTheDesigner}",
```
Comment (balajikris) [inline src/EditorFeatures/VisualBasicTest/EndConstructGeneration/MethodBlockTests.vb]: where does this resource come from?

File: src/EditorFeatures/VisualBasicTest/Diagnostics/Suppression/SuppressionTests.vb [hunk 2]
```diff
@@ -69,17 +69,17 @@ Class C
         [|Dim x As Integer|]
     End Sub
 End Class]]>
-                    Dim expected = <![CDATA[
+                    Dim expected = $"
 Imports System
 Class C
     Sub Method()
-#Disable Warning BC42024 ' Unused local variable
+#Disable Warning BC42024 ' {WRN_UnusedLocal_Title}
```
Comment (balajikris) [inline src/EditorFeatures/VisualBasicTest/Diagnostics/Suppression/SuppressionTests.vb]: Where does this resource come from?

File: src/EditorFeatures/VisualBasicTest/EndConstructGeneration/MethodBlockTests.vb (hunk 1 above)
Comment (brettfo) [inline src/EditorFeatures/VisualBasicTest/EndConstructGeneration/MethodBlockTests.vb]: It's from VBEditorResources, but the `VBEditorResources.` prefix is unnecessary.

File: src/EditorFeatures/VisualBasicTest/Diagnostics/Suppression/SuppressionTests.vb (hunk 2 above)
Comment (brettfo) [inline src/EditorFeatures/VisualBasicTest/Diagnostics/Suppression/SuppressionTests.vb]: From VBResources.  Similar case as the other, `VBResources.` is unnecessary.

**Pull request merged**